README.md

```diff
@@ -1,7 +1,141 @@
 
-***Mattermost Boards** is installed and enabled by default in Mattermost v6.0 and later.*
+![Focalboard](website/site/static/img/hero.jpg)
+
+## Try Focalboard
+
+### Mattermost Plugin
+
+After downloading and installing the plugin in the System Console, select the menu in the top left corner and select **Boards**. Access the latest releases of the focalboard plugin by downloading the `mattermost-plugin-focalboard.tar.gz` file from the releases in this repository: <https://github.com/mattermost/focalboard/releases>
+
+
+### API Docs
+
+Boards API docs can be found over at <https://htmlpreview.github.io/?https://github.com/mattermost/focalboard/blob/main/server/swagger/docs/html/index.html>
+
+### Getting started
+
+Our [developer guide](https://developers.mattermost.com/contribute/more-info/focalboard/mattermost-boards-setup-guide/) has detailed instructions on how to set up your development environment for the **Mattermost boards plugin**. You can also join the [~Focalboard community channel](https://community.mattermost.com/core/channels/focalboard) to connect with other developers.
+
+Clone [mattermost-server](https://github.com/mattermost/mattermost-server) into sibling directory.
+
+You also want to have the environment variable `MM_DEBUG"true"` set, otherwise the plugin
+will be compiled for Linux, Windows, and Darwin ARM64 and x64 architecture every single time. Setting
+the `MM_DEBUG` to `true` makes the plugin compile and build only for the OS and architecture
+you are building on.
+
+In your Mattermost configuration file, ensure that `PluginSettings.EnableUploads` is set to `true`, and `FileSettings.MaxFileSize` is
+set to a large enough value to accept the plugin bundle (eg `256000000`).
+
+### Building the plugin
+
+Run the following command in the plugin repository to prepare a compiled, distributable plugin ZIP file:
+
+```bash
+make dist
+```
+
+After a successful build, a `.tar.gz` file in the `/dist` folder will be created which can be uploaded to Mattermost. To avoid having to manually install your plugin, deploy your plugin using one of the following options.
+
+##### Building in Dev Mode
+
+Set the following environment variables to true before running `make dist`-
+
+1. MM_DEBUG
+
+### Deploying with Local Mode
+
+If your Mattermost server is running locally, you can
+enable [local mode](https://docs.mattermost.com/administration/mmctl-cli-tool.html#local-mode) to streamline deploying
+your plugin. Edit your server configuration as follows:
+
+```
+{
+    "ServiceSettings": {
+        ...
+        "EnableLocalMode": true,
+        "LocalModeSocketLocation": "/var/tmp/mattermost_local.socket"
+     }
+}
+```
+
+and then deploy your plugin:
+
+```bash
+make deploy
+```
+
+If developing a plugin with a web app, watch for changes and deploy those automatically:
+
+```bash
+export MM_SERVICESETTINGS_SITEURL=http://localhost:8065
+make watch-plugin
+```
+
+## How to Release
```

Comment:
Is this necessary? @marianunez

Reply:
@esethna This was one of the suggestions from the delivery team. [See this thread](https://hub.mattermost.com/private-core/pl/sn1dt49obb833ek1d8c3eiuk9w)

Reply:
+1 - this is useful for internal documentation